src/averageAges.js

```diff
@@ -14,12 +14,24 @@
  *
  * @return {number}
  */
+const MALE = 'm';
+const FEMALE = 'f';
+const ONE_CENTURY = 100;
+
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const menInCentury = people
+    .filter(person => person.sex === FEMALE ? false
+      : !century ? true
+        : Math.ceil(person.died / ONE_CENTURY) === century);
```

Comment:
Now it looks like: 
```js
if (person.sex === FEMALE) {
  return false;
} else {
  if (!century) {
    return true;
  } else {
    return Math.ceil(person.died / ONE_CENTURY) === century;
  }
}
```

I think we can make your condition a little bit shorter.


little tip:
`person.sex === "m" & (century === true ? do something : do something)`